bin/setup_psqlgraph.py

```diff
@@ -80,22 +80,22 @@ def create_indexes(host, user, password, database):
     index = lambda t, c: ["CREATE INDEX ON {} ({})".format(t, x) for x in c]
     for scls in Node.get_subclasses():
         tablename = scls.__tablename__
-        map(engine.execute, index(
+        list(map(engine.execute, index(
             tablename, [
                 'node_id',
-            ]))
-        map(engine.execute, [
+            ])))
+        list(map(engine.execute, [
```

Comment:
list() unnecessary

Reply:
The lists on the `map()` function calls, I do need to force it to execute https://github.com/uc-cdis/sheepdog/pull/314#discussion_r362079395

Reply:
My bad!